Servers/UI/OJS.Servers.Ui/ClientApp/src/utils/urls.ts

```diff
@@ -307,4 +323,6 @@ export {
     getSubmissionsUrl,
     encodeUsernameAsUrlParam,
     decodeUsernameFromUrlParam,
+    getContestTypeResultsUrl,
+    getContestBtnUrlString,
 };
```

Comment:
Again the naming seems off. One ends with `Url`, the other with `UtlString`.
getContestTypeResultsUrl => getContestResultsUrl. This `Type` does not make much sense in the name IMO.
getContestBtnUrlString => getContestParticipateUrl

Reply:
It has type since it determines the url based on participationType. I dont really have an idea for other names for those funcitons. Im open for suggestions :D. There is only one contest related button and he is in his own component -> ContestButton, so I think that contestBtn in the function makes sense, anyway, Im open for name suggestion here as well

Reply:
I have suggested some names for both. Doesn't matter that there is only one contest button. These util functions does not have notion where they are used and should not take this into account. The name should reflect the purpose of the url.

Reply:
done